packages/react-instantsearch/examples/react-router/src/App.js

```diff
@@ -0,0 +1,99 @@
+import React, {Component} from 'react';
+import {
+    InstantSearch, HierarchicalMenu,
+    Hits, Menu, Pagination, PoweredBy, RangeRatings,
+    RefinementList, SearchBox, ClearAll,
+} from 'react-instantsearch/dom';
+/* eslint-disable import/no-unresolved */
```

Comment:
Seems like we have some eslint issues here, we could solve them I'll have a look afterwards